src/Mod/Sketcher/Gui/CommandSketcherBSpline.cpp

```diff
@@ -407,9 +407,9 @@ bool CmdSketcherCompBSplineShowHideGeometryInformation::isActive(void)
 }
 
 // Convert to NURBS
-DEF_STD_CMD_A(CmdSketcherConvertToNURB)
+DEF_STD_CMD_A(CmdSketcherConvertToNURBS)
 
-CmdSketcherConvertToNURB::CmdSketcherConvertToNURB()
+CmdSketcherConvertToNURBS::CmdSketcherConvertToNURBS()
     : Command("Sketcher_BSplineConvertToNURB")
```

Comment:
> `Sketcher_BSplineApproximate` name is only used for the icon name in file.

The `WhatsThis` entry points to Sketcher_BSplineConvertToNURB which auto-redirects to Sketcher_BSplineApproximate

So you're proposing the wiki page should be [Sketcher_BSplineConvertToNURB](https://wiki.freecad.org/Sketcher_BSplineConvertToNURB) should be un-redirected and renamed to Sketcher_BSplineConvertToNURBS ?

Reply:
Not sure about that. But possibly. That is the name we use in code, and NURBS is more common than NURB.

Reply:
Oh I see. Should be fixed now.